tests/lib/rules/no-missing-import.js

```diff
@@ -58,6 +58,14 @@ ruleTester.run("no-missing-import", rule, {
             filename: fixture("test.js"),
             code: "import a from './a.js';",
         },
+        {
+            filename: fixture("test.ts"),
+            code: "import a from './a.js';",
+        },
+        {
+            filename: fixture("test.ts"),
+            code: "import a from './d.ts';",
```

Comment:
not sure I got the point - shouldn't it be `./d.js`?

Reply:
yes, you are right about that. That would have failed a tsc build.